web/src/app/shared/util/taskana.date.ts

```diff
@@ -17,11 +17,15 @@ export class TaskanaDate {
   }
 
   public static getDateToDisplay(date: string): string {
+    return this.transformDateWithTimeZone(date);
+  }
+
+  public static transformDateWithTimeZone(date: string): string | null {
     const dateFormat = 'yyyy-MM-dd HH:mm:ss';
     const dateLocale = 'en-US';
     const datePipe = new DatePipe(dateLocale);
 
-    return datePipe.transform(date, dateFormat);
+    return datePipe.transform(date, dateFormat, Intl.DateTimeFormat().resolvedOptions().timeZone);
   }
 
   public static convertToBrowserTimeZone(date: Date): string {
```

Comment:
checkout this method please, usages is it usefull, duplicated code?

Reply:
I remove the convertToBrowserTimeZone method because it is never used